Sprint-3/1-key-implement/1-get-angle-type.js

```diff
@@ -9,6 +9,12 @@
 
 function getAngleType(angle) {
    if (angle === 90) return "Right angle";
+   else if (angle > 0 && angle < 90) return "Acute angle";
+   else if (angle > 90 && angle < 180) return "Obtuse angle";
+   else if (angle === 180) return "Straight angle";
+   else if (angle > 180 && angle < 360) return "Reflex angle";
+   
+   
    // read to the end, complete line 36, then pass your test here
```

Comment:
Excellent. One thing you could look into as an extension exercise is to think about what might happen if `angle` were not a number. JavaScript has functionality for checking this and returning an error that you could learn about, and using it would make your code more robust in the real world.